### ISSUE TITLE
Colorize labels

### ISSUE THREAD

**Issue by GlancingMind** [GlancingMind/git-bug#9]
Current all labels have the same color. It may be beneficial to assign (unique)
colors to a label (manually and automatically). Depending on the backend, this may require Go knowledge.

**Comment by MichaelMure** [comment]
The label's color is exposed in the graphQL API.

**Comment by GlancingMind** [comment]
Oh, the description also seems outdated (created the issues by looking at the 7.0.1 release WebUI).
The labels already have different colors. This resolves one part of the issue.
Lets build another page to change existing labels color and description.

**Comment by MichaelMure** [comment]
> created the issues by looking at the 7.0.1 release WebUI

Yeah, be careful about that, a lot has changed since that release.

> Lets build another page to change existing labels color and description.

The problem is that there is nothing to store the label color in the core yet. That would exist if we have a project settings data-structure someday but we are not there yet. At the moment the label's color are generated based on a hash of their string. I believe the webUI already respect those colors? If so, there is nothing to do here IMHO.
